src/Common/RJSF/widgets/Select.tsx

```diff
@@ -15,7 +15,6 @@
  */
 
 import React from 'react'
-import isEqual from 'lodash/isEqual'
```

Comment:
This performs deep equal check, check if needed

Reply:
Yes deep equals is required here and therefore using deepEqual util function from common-lib itself instead of lodash. Verified the change